detox/e2e/test/messaging/tap_send_button_repeatedly.e2e.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import nodeExpect from 'expect';
+
+import {toChannelScreen} from '@support/ui/screen';
+import {Channel, Post, Setup} from '@support/server_api';
+
+describe('Messaging', () => {
+    let team;
+    let user;
+
+    beforeAll(async () => {
+        ({team, user} = await Setup.apiInit());
+
+        await toChannelScreen(user);
+    });
+
+    it('MM-T109 User can\'t send the same message repeatedly', async () => {
+        const message = Date.now().toString();
+
+        // # Type a message
+        const postInput = await element(by.id('post_input'));
+        await postInput.tap();
+        await postInput.typeText(message);
+
+        // # Tap the send button
+        const sendButton = await element(by.id('send_button'));
+        await expect(sendButton).toBeVisible();
+        await sendButton.tap();
+        await expect(sendButton).not.toExist();
+
+        // # Then tap send button repeatedly
+        const disabledSendButton = await element(by.id('disabled_send_button'));
+        await expect(disabledSendButton).toBeVisible();
+        await expect(disabledSendButton).toExist();
+        await disabledSendButton.tap();
```

Comment:
Sure, updated. It's better to read now using `multiTap`. It would be nice if it can be done on `send_button`. However, it's not possible since after the first tap, it's being replaced with a different `disabled_send_button` component.